TheaterDatabase/dbo/Tables/MainImage.sql

```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[MainImage] (
```

Comment:
Why we need special table for mainimages? 
it's 1 to 1 relation with Performance
Better just add field image to Performance